frontend/src/store/modules/users/actions.js

```diff
@@ -29,19 +25,17 @@ export default {
                     resolve(result);
                 })
                 .catch((error) => {
-                    payload.failCallback();
+                    if (result.status !== 200) {
```

Comment:
result.status can't be 200 in catch block by default, i think just payload.failCallback(); should be enough